components/opponent/wikis/fighters/opponent_custom.lua

```diff
@@ -32,13 +33,56 @@ function CustomOpponent.readOpponentArgs(args)
 
 	local partySize = Opponent.partySize(opponent.type)
 
-	if partySize == 1 then
-		opponent.players[1].chars = Array.parseCommaSeparatedString(args.chars)
-		opponent.players[1].game = args.game or Variables.varDefault('tournament_game') or Info.defaultGame
-	elseif partySize then
-		Array.forEach(opponent.players, function (player, playerIndex)
-			player.chars = Array.parseCommaSeparatedString(args['chars' .. playerIndex])
+	if not partySize then return opponent end
+
+	local game = args.game or Variables.varDefault('tournament_game') or Info.defaultGame
+	opponent.players[1].game = game
```

Comment:
Do we need to store the game on all of the players?

Reply:
old one did neither
only thing i hope to achieve here is to standardize the inputted chars by pushing them through the data modules
(would have used the function from MGI input util, but then i would get circular requires)